tests/test_bjson.js

```diff
@@ -285,10 +285,10 @@ function bjson_test_bytecode()
 function bjson_test_fuzz()
 {
     var corpus = [
-        "FBAAAAAABGA=",
```

Comment:
Note: this was complaining because of the bytecode version, but I don't know how the corpus was generated so I just incremented it manually...